src/index.html

```diff
@@ -1,12 +1,399 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+  <header class="header">
+    <a href="#">
+      <img
+        src="images/logo.png"
+        class="header__logo"
+        alt="logo"
+      >
+    </a>
+    <nav class="nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+          <a href="#" class="nav__link is-active">Apple</a>
```

Comment:
when working with BEM, it is cool to use modifiers
```suggestion
          <a href="#" class="nav__link nav__link--active">
            Apple
          </a>
```